src/qml/pages/MainPage.qml

```diff
@@ -175,4 +202,15 @@ Page {
             controllers.fileCloud.addFile(url)
         }
     }
+
+    function textClipper(text, textPlaceholderWidth, textContentWidth) {
```

Comment:
This fix looks complicated.
Can it be done without extra function, just with pure text eliding & anchors?

Reply:
This fix adds functionality to trim a string and add three dots to the end of a string. There is no simpler method. I can try to make the code more concise.

Reply:
So why text eliding can't be used instead of this?

Reply:
My bad. I'll fix